### PR TITLE
Docs for Netlify Image CDN implementation

### PR DESCRIPTION
**Pull request by klavavej**
This PR adds docs for the Netlify Image CDN implementation proposed in https://github.com/nuxt/image/pull/1234

Very open to feedback as this is my first OSS contribution and I'm guessing a bit at what info is necessary to include in the Nuxt docs (as opposed to being made available through links to the Netlify docs) by looking at the docs for other providers.

### CODE REVIEW

File: docs/content/3.providers/netlify.md [hunk 1]
```diff
@@ -8,21 +8,68 @@ links:
...
+```toml [netlify.toml]
+[images]
+  remote_images = ["https://my-images.com/.*", "https://animals.more-images.com/[bcr]at/.*"]
+```
+
+The `remote_images` property accepts an array of regex. If your images are in specific subdomains or directories, you can use regex to allow just those subdomains or directories.
+
+## Modifiers
+
+In addition to the [standard modifiers](https://image.nuxt.com/usage/nuxt-img#modifiers), all of the [Netlify Image CDN transformation options](https://docs.netlify.com/image-cdn/overview/#transform-images) are available as modifiers for Nuxt Image.
```
Comment (klavavej) [inline docs/content/3.providers/netlify.md:36]:
```suggestion
In addition to the [standard properties](https://image.nuxt.com/usage/nuxt-img), all of the [Netlify Image CDN transformation options](https://docs.netlify.com/image-cdn/overview/#transform-images) are available as modifiers for Nuxt Image.
```

File: docs/content/3.providers/netlify.md [hunk 2]
```diff
@@ -8,21 +8,68 @@ links:
...
+  provider="netlify"
+  src="owl.jpg"
+  :modifiers="{ 
+    h="400"
+    w="600"
+    fit="cover"
+    position="left"
+    fm="webp"
+    q="80"
+  }"
```
Comment (ascorbic) [inline docs/content/3.providers/netlify.md]: Only position is a non-standard property. The rest are standard. e.g. see https://image.nuxt.com/usage/nuxt-img#modifiers

**Comment by ascorbic** [comment]
It will automatically enabled when deploying to Netlify, so we could add something along the lines of "this provider is automatically enabled when deploying to Netlify, however you can also choose it manually". We should also say soemthing about how if they've manually enabled it, then for local development they will need to use the Netlify CLI to see images.

**Comment by klavavej** [comment]
>  however you can also choose it manually

Is this applicable for sites that aren't deployed on Netlify? If no, why would someone choose to manually enable the netlify provider instead of relying on automatic enablement?

- - - - 

Update - the more I think about this the less my initial understanding makes sense. I'm now assuming manual enablement is useful for local development more than for sites that aren't deployed on Netlify. lemme know if I'm still misunderstanding

**Comment by klavavej** [comment]
Thanks for the feedback - ready for another review!

**Comment by ascorbic** [comment]
I've made a couple of changes and will now merge this into my PR

**Pull request merged**